ssh/tailssh/tailssh.go

```diff
@@ -234,6 +234,8 @@ type conn struct {
 	userGroupIDs []string     // set by doPolicyAuth
 	acceptEnv    []string
 
+	clientAuthAttempted atomic.Bool
```

Comment:
why the atomic? Doesn't the go ssh package call into us from a single goroutine?

Reply:
It does, I mostly chose it for the CompareAndSwap semantics. Will change.